general/mem_manager.hpp

```diff
@@ -182,14 +193,25 @@ class Memory
    /// Copy constructor: default.
    Memory(const Memory &orig) = default;
 
-   /// Move constructor: default.
-   Memory(Memory &&orig) = default;
+   /** Move constructor. Sets the pointers and associated ownership of validity
+       flags of @a *this to those of @a other. Invalidates @a other. */
```

Comment:
Instead of "Invalidates", I'd just say "Resets" or "Clears the contents of".

Reply:
done in [418054b](https://github.com/mfem/mfem/pull/2355/commits/418054b770a4cac104a33b46f18dce432a3f5482)